documentation/general/dash-sonic-hld.md

```diff
@@ -494,8 +495,25 @@ vm_vni                   = VM VNI that is used for setting direction. Also used
 
 ### 3.2.9 ROUTE LPM TABLE - OUTBOUND
 
+```
+DASH_ENI_ROUTE_TABLE:{{eni}}:
```

Comment:
nit:
- remove the last ":"
- update eni to "eni_id"

Reply:
removed `:`, we use `eni `in other places, so IMO, we can stick to the same format